run_scripts_testing/ice_shelf_cavity/isomip_plus/isomip_plus_3d_extruded_layered.py

```diff
@@ -809,10 +811,10 @@ def solve(self):
 ##########
 
 # Set up Vectorfolder
-folder = "/rds/general/user/wis15/home/data/3d_isomip_plus/extruded_meshes/"+str(args.date)+"_3d_isomip+_dt"+str(dt)+\
-         "_dtOut"+str(output_dt)+"_T"+str(T)+"_StratLinTres"+str(restoring_time.values()[0])+\
-         "_Muh"+str(mu_h.values()[0])+"_switchMuv"+"_Kh"+str(kappa_h.values()[0])+"_switchKv"+\
-         "_dx"+str(round(1e-3*dy))+"km_lay"+str(args.nz)+"_closed_coriolis_tracerlims_ip3_alignicefront_backeul_fromrest_switchP0dg/" #_smagviscmax"+str(max_nu)+"_officeshelf/"
+folder = '.' #"/rds/general/user/wis15/home/data/3d_isomip_plus/extruded_meshes/"+str(args.date)+"_3d_isomip+_dt"+str(dt)+\
+         #"_dtOut"+str(output_dt)+"_T"+str(T)+"_StratLinTres"+str(restoring_time.values()[0])+\
+         #"_Muh"+str(mu_h.values()[0])+"_switchMuv"+"_Kh"+str(kappa_h.values()[0])+"_switchKv"+\
+         #"_dx"+str(round(1e-3*dy))+"km_lay"+str(args.nz)+"_closed_coriolis_tracerlims_ip3_alignicefront_backeul_fromrest_switchP0dg/" #_smagviscmax"+str(max_nu)+"_officeshelf/"
```

Comment:
If you prefer to keep it there, I don't mind - but generally speaking, I'd avoid committing commented out stuff

Also use format strings! i.e. `f"..._3d_isomip+_dt{dt}_dtOut{output_dt}_T{T}_StratLinTres..."`

Reply:
I'll get rid of the comment! I will try and come back to this script to make it tidier